.github/workflows/test.yml

```diff
@@ -0,0 +1,100 @@
+name: Test
+
+on:
+  push:
+  pull_request:
+
+env:
+  GO111MODULE: on
+
+jobs:
+
+  golangci-lint:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+
+      - uses: actions/checkout@v2
+
+      - uses: golangci/golangci-lint-action@v2
+        with:
+          version: latest
+          args: --verbose
+
+  test-unix:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+        - ubuntu
+        - macOS
+        go:
+        - 14
+        - 15
+        - 16
+        - 17
```

Comment:
I would recommend that you also include Windows in the matrix and reduce the numbers by cutting one or two Go versions.

Reply:
We do test against windows here: 

https://github.com/spf13/cobra-cli/blob/7c6e26ac0939450323b7dd4acf8d8280dde3879e/.github/workflows/test.yml#L66

And yes, will remove Go 14 and 15.